### DISCUSSION TITLE
template provisioning

### DISCUSSION THREAD

**Discussion by wibed** [lima-vm/lima#2973] [Q&A]
is there something like a registry that provisions templates. community driven or other.
there has to be an interest in templates of all kinds.

**Comment by AkihiroSuda** [comment]
HTTPS URL can be passed to `limactl create` (use carefully, with a trustable source)
```
limactl create https://raw.githubusercontent.com/lima-vm/lima/master/templates/alpine.yaml
```

**Comment by afbjorklund** [comment]
It is also possible to add a `lima.yaml` file to the project itself, similar to a `Vagrantfile`.

e.g. https://github.com/afbjorklund/nerdctld/tree/main/lima

**Comment by jandubois** [comment]
I think @wibed was asking about some kind of registry instead; something you could search for examples etc. like you have [ArtifactHUB](https://artifacthub.io) for Helm charts, [Docker Hub](https://hub.docker.com) for container images, or [Hugging Face](https://huggingface.co) for AI models.

It would be nice to have something like that, but it would require somebody to own/maintain it.

I hoped that Gists had "topics", like GitHub repos, then we could start a convention with `lima-template` label, but alas, they don't. Some kind of self-organizing mechanism like this would be a good starting point (anything that doesn't require the Lima maintainers to manage/approve contributions).

For curiosity I did a [GitHub search](https://github.com/search?type=code&q=path%3A**%2Flima.yaml) for `path:**/lima.yaml`, which might be a starting point to look for other examples.
